docs/auditLogs.adoc

```diff
@@ -0,0 +1,167 @@
+* `system.security.principal.delete` - a principal (user, group or role) has been deleted
+* `system.security.principal.addRelationship` - a relationship has been added to principal
+* `system.security.principal.removeRelationship` - a relationship has been removed.
+* `system.security.principal.removeRelationships` - all relationships have been removed
+* `system.security.principal.setPassword` - a password has been set for a user
+* `system.security.idProvider.create` - an ID provider has been created
+* `system.security.idProvider.update` - an ID provider has been updated
+* `system.security.idProvider.delete` - an ID provider has been deleted
+
+Below you can see the examples how data stores for actions with the following types:
+
+.`system.security.principal.addRelationship`
+
+User `user:system:username` has been added to group `group:system:usergroup`.
+[source,JSON]
+----
+{
+  "type": "system.security.principal.addRelationship",
+  "source": "com.enonic.xp.security",
+  "user": "user:system:admin",
+  "objects": ["user:system:username","group:system:usergroup"],
+  "data": {
+    "params": {
+      "principal": "user:system:username",
+      "joins": "group:system:usergroup"
+    }
+  }
+}
+----
+
+Role `role:system.admin` has been assigned to user `user:system:username`.
+[source,JSON]
+----
+{
+  "type": "system.security.principal.addRelationship",
+  "source": "com.enonic.xp.security",
+  "user": "user:system:admin",
+  "objects": ["user:system:username","role:system.admin"],
+  "data": {
+    "params": {
+      "principal": "user:system:username",
+      "joins": "role:system.admin"
+    }
+  }
+}
+----
+
+Group `group:system:group1` has been added to group `group:system:group2`.
+[source,JSON]
+----
+{
+  "type": "system.security.principal.addRelationship",
+  "source": "com.enonic.xp.security",
+  "user": "user:system:admin",
+  "objects": ["group:system:group1","group:system:group2"],
+  "data": {
+    "params": {
+      "principal": "group:system:group1",
+      "joins": "group:system:group2"
+    }
+  }
+}
+----
+
+.`system.security.principal.removeRelationship`
+
+Role `role:system.admin` has been unassigned from user `user:system:username`.
+[source,JSON]
+----
+{
+  "type": "system.security.principal.removeRelationship",
+  "source": "com.enonic.xp.security",
+  "user": "user:system:admin",
+  "objects": ["user:system:username","role:system.admin"],
+  "data": {
+    "params": {
+      "principal": "user:system:username",
+      "leaves": "role:system.admin"
+    }
+  }
+}
+----
+
+.`system.security.principal.removeRelationships`
+
+Remove all relationships from `group:system:usergroup`.
+[source,JSON]
+----
+{
+ "type": "system.security.principal.removeRelationships",
+  "source": "com.enonic.xp.security",
+  "user": "user:system:admin",
+  "objects": "group:system:usergroup",
+  "data": {
+    "params": {
+      "from": "group:system:usergroup"
+    }
+  }
+}
+----
```

Comment:
Not sure it should be documented. It only appears when deprecated method is called